### DISCUSSION TITLE
Multi User Editor

### DISCUSSION THREAD

**Discussion by sardarms** [sveltia/sveltia-cms#94] [Ideas]
I am not sure if the feature exists, but I could not find it. 

What I would like is to have additional users, who will be able to add new files/images and post and change already existing files. 

Thank you :-)

**Comment by kyoshino** [comment]
User management is done through the access control of a GitHub repository. Basically, all you need to do is to invite people to your repository. Anyone with commit access can use Sveitla CMS to add/edit entries and upload assets 🙂

At this moment there is no way to assign roles to people like in WordPress or some other CMSes. I plan to add the feature later this year.

**Reply by rdeets** [reply]
Will this be added soon?

**Reply by kyoshino** [reply]
I understand that there are needs, but since there are still many basic features missing, I’ll work on it after the 1.0 release, sometime next year. FYI, these are relevant Netlify/Decap CMS issues:

- https://github.com/decaporg/decap-cms/issues/2 — Very old issue
- https://github.com/decaporg/decap-cms/issues/277 — Need a server-side gimmick. I created a similar POC using Firebase Realtime Database at my former company
- https://github.com/decaporg/decap-cms/issues/7303 — Filed a few days ago

**Reply by rdeets** [reply]
Okay thank you for your hard work bro
